lib/elixir/src/elixir_def_overridable.erl

```diff
@@ -32,6 +32,13 @@ name(_Module, {Name, _} = Function, Overridable) ->
   {Count, _, _, _} = maps:get(Function, Overridable),
   elixir_utils:atom_concat([Name, " (overridable ", Count, ")"]).
 
+%% Returns the kind (def, defp, and so on) of the given overridable function.
+
+kind(Module, FunctionOrMacro) ->
```

Comment:
Maybe we can rename the `name` function to `kind_and_name` and return both kind and name?